lib/blocks/sourcemap.js

```diff
@@ -0,0 +1,21 @@
+module.exports = blockConfig => (processEnv, argv) => config => {
+    const isDev = processEnv.NODE_ENV === "development";
+
+    const defaultConf = {
+        devtool: isDev ? "cheap-eval-source-map" : "none",
```

Comment:
woudn't `sourceMap` be a better name ?

Reply:
I don' think so, this is relative to https://webpack.js.org/configuration/devtool/

Reply:
Okay din't know